cob_script_server/src/simple_script_server/simple_script_server.py

```diff
@@ -355,26 +359,29 @@ def move_base(self,component_name,parameter_name,blocking, mode):
 
 		# check trajectory parameters
 		if not type(param) is list: # check outer list
-				rospy.logerr("no valid parameter for %s: not a list, aborting...",component_name)
-				print "parameter is:",param
-				ah.set_failed(3)
-				return ah
+			message = "No valid parameter for " + component_name + ": not a list, aborting..."
+			rospy.logerr(message)
+			print "parameter is:",param
```

Comment:
I know this has been like this before, but shouldn't this rather be a `logwarn` (or `loginfo`)?